src/dmd/func.d

```diff
@@ -2615,9 +2615,14 @@ extern (C++) FuncDeclaration resolveFuncCall(const ref Loc loc, Scope* sc, Dsymb
                 }
                 else
                 {
+                    auto fullFdPretty = fd.toPrettyChars();
                     .error(loc, "%smethod `%s` is not callable using a %sobject",
-                        funcBuf.peekString(), fd.toPrettyChars(),
+                        funcBuf.peekString(), fullFdPretty,
                         thisBuf.peekString());
+                    if (funcBuf.peekSlice() == "mutable ")
```

Comment:
Isn’t there a better way to do this than comparing strings?

Reply:
Then I would have to duplicate this logic:

https://github.com/dlang/dmd/blob/be225ecd2d7dab2dfa878e3ed2c2fc0b39edbdab/src/dmd/func.d#L2439-L2461

Reply:
though `MODMatchToBuffer` returns `void` and I could modify it to return something more useful like:

```d
struct Errors {
    bool isNotShared;
    bool isMutable;
}
```

Reply:
Yes, exactly.

Reply:
OK done.